pkgs/development/python-modules/generic/default.nix

```diff
@@ -129,6 +129,11 @@ python.stdenv.mkDerivation (builtins.removeAttrs attrs ["disabled" "doCheck"] //
   # Python packages don't have a checkPhase, only an installCheckPhase
   doCheck = false;
 
+  passthru = {
```

Comment:
Two off the top of my head:

* package version, this should be a required argument to `buildPythonPackage` as opposed to wrapping it in the name anyway.
* path to the `site-packages` folder of `self`


Probably a few others I'm missing :)